README.md

```diff
@@ -127,6 +127,14 @@ Comet shuffle feature is disabled by default. To enable it, please add related c
 Above configs enable Comet native shuffle which only supports hash partiting and single partition.
 Comet native shuffle doesn't support complext types yet.
 
+Depending on your deployment mode you may also need to set the driver & executor class path(s) to explicitly contain Comet otherwise Spark may use a different class-loader for the Comet components than its internal components which will then fail at runtime. For example:
+
+```
+--driver-class-path spark/target/comet-spark-spark3.4_2.12-0.1.0-SNAPSHOT.jar
```

Comment:
Maybe also add executor configuration alongside? Although it is trivial but as we already go to update this.